tml_decoder/encoders/mini_lm_encoder.py

```diff
@@ -11,15 +11,18 @@ def __init__(self):
     def similarity(self, veca, vecb):
         return veca@vecb
     
-    def average_embedding(self, embeddings):
-        stacked = np.stack(embeddings)
-        emb_sum = np.sum(stacked, axis=0)
-        return emb_sum/np.linalg.norm(emb_sum)
+    def average_embedding(self, embeddings: np.ndarray) -> np.ndarray:
+        emb_sum = np.sum(embeddings, axis=0)
+        return emb_sum / np.linalg.norm(emb_sum)
     
-    def encode(self, text: str):
-        return self.encoder.encode(text)
+    def encode(self, text: str) -> np.ndarray:
+        result = self.encoder.encode(text)
+        if isinstance(result, np.ndarray):
+            return result
+        
+        raise TypeError("Expected ndarray from self.encoder.encode, got {}".format(type(result)))
     
     @property
     def name(self):
-        return "Mini LM"
+        return "gtr-base"
```

Comment:
to nie jest gtr-base

Reply:
xDD

Reply:
przepraszam coś mi się kliknęło i nie wiem jak to cofnąć